Model 13.3: Update to end-of-ENN prevalence for preterm in LNN CSMRisk equation (#1732)
ihmeuw/vivarium_research: docs/source/models/causes/neonatal/preterm_birth.rst

  It is even possible that there is no way to make this work consistently, meaning that any choice of weight function would lead to negative mortality risks.  We expect that this will not be an issue, but we haven't actually tried it with the real data yet.

:math:`\text{CSMRisk}` and :math:`p_{\text{preterm}}` are calculated differently for the ENN and LNN age groups.
For clarity of notation, in what follows we will again make explicit the age group
subscripts that have been implicit on every quantity to this point.
(Sex and location remain implicit.)
We define the ENN CSMRisk as:

.. math::

  \text{CSMRisk}_\text{ENN} = \frac{\text{enn_death_count}}{\text{live_birth_count}}

where the :math:`\text{enn_death_count}` and :math:`\text{live_birth_count}` are
quantities pulled from GBD, as detailed in the table below.

The LNN CSMRisk is:

.. math::

>   \text{CSMRisk}_\text{LNN} = \frac{\text{lnn_death_count}}{\text{live_birth_count} - \text{enn_all_cause_death_count}}
> 
> :math:`p_{\text{preterm}}`, as mentioned above, represents the prevalence/exposure

Reviewer: ```suggestion
  \text{CSMRisk}_\text{LNN} = \frac{\text{lnn_death_count}}{\text{live_birth_count} - \text{enn_all_cause_death_count}},

where, again, all quantities are pulled from GBD as detailed in the table below.

:math:`p_{\text{preterm}}`, as mentioned above, represents the prevalence/exposure
```

This was a bit confusing when I first read it because there was no period after the equation, but I think $p\_{\text{preterm}}$ is the start of a new sentence (and maybe a new paragraph?). Also, I think it will make things clearer to specify that these quantities also come from GBD like in the previous equation.

of preterm (gestational age < 37 weeks) at the *beginning* of the age group.
For ENN, the beginning of the age group is birth, so the prevalence of preterm
at birth is a sum of the birth prevalence for all LBWSG categories with gestational
age less than 37 weeks:

.. math::

  p_{\text{preterm},\text{ENN}} = \sum_{\{\text{cat}: \text{GA}<37\}} \text{lbwsg_birth_prevalence}_\text{cat}

where :math:`\text{lbwsg_birth_prevalence}` can be pulled from GBD with minor transformations,
as detailed in the table below.

> For LNN the situation is more complicated, because we need to account

Reviewer 2: ```suggestion
For LNN, the situation is more complicated, because we need to account
```

for differential mortality in the ENN period.
Therefore, the easiest way to calculate :math:`p_{\text{preterm},\text{LNN}}` is to get the end-of-ENN preterm
prevalence from the same LBWSG PAF calculation pipeline used for :math:`Z`
above.
As detailed at :ref:`details_of_the_lbwsg_paf_calculation` on the neonatal all-cause
mortality page, there are two iterative steps using microsimulation, with the late neonatal calculations
using the result of the early neonatal calculations.
Similarly to the LNN PAF, *after* the early neonatal calculations are complete, the prevalence of
preterm at the end of the ENN age group should be calculated.
This value should be used as :math:`p_{\text{preterm},\text{LNN}}` for the purposes
of the CSMRisk equation.

> Concretely, in the PAF calculation pipeline, in which LBWSG categories have not been assigned using the birth prevalence (currently exactly equally between categories),

Reviewer: I find this sentence confusing... Is this referring to the fact that in the PAF calculation sim, we initialize each category with the same number of simulants, rather than initializing each category with a number of simulants proportional to its birth prevalence? Is that documented somewhere? There must be a clearer way to get this across, maybe some way that explicitly mentions the "grid points" referred to parenthetically after the equation...

Author: Yes it is, I will revise to clarify!

the prevalence is calculated as follows:

.. math::

  p_{\text{preterm},\text{LNN}} = \frac{

>     \sum_{\{\text{cat}: \text{GA}<37\}} \text{lbwsg_birth_prevalence}_\text{cat} \times \frac{n_\text{cat} - n^\text{deaths}_\text{cat}}{n_\text{cat}}

Reviewer 3: Should this (and other instances) be less than or equal to instead of less than?

  }{
    \sum_{\text{cat}} \text{lbwsg_birth_prevalence}_\text{cat} \times \frac{n_\text{cat} - n^\text{deaths}_\text{cat}}{n_\text{cat}}
  }

> where :math:`n_\text{cat}` is the number of simulants initialized into each category before mortality was applied (the number of grid points in each category) and :math:`n^\text{deaths}_\text{cat}` is the number of deaths in each category when ENN mortality was applied in the PAF calculation pipeline. Note that :math:`n_\text{cat}` will not vary by LBWSG exposure category under the current approach of assigning LBWSG exactly equally between categories.

Reviewer: The phrase "assigning LBWSG exactly equally between categories" seems imprecise/unclear to me. Maybe:

```suggestion
where :math:`n_\text{cat}` is the number of simulants initialized into each category before mortality was applied (the number of grid points in each category) and :math:`n^\text{deaths}_\text{cat}` is the number of deaths in each category when ENN mortality was applied in the PAF calculation pipeline. Note that :math:`n_\text{cat}` will not vary by LBWSG exposure category under the current approach of calculating a weighted average after assigning the same number of simulants to each LBWSG category.
```

Author: Thanks! I'm going to propagate this idea a bit more into the preceding paragraph.
